docs/composer-studio/developer-guides/guides/remix.md

```diff
@@ -0,0 +1,256 @@
+# Remix
+
+[Remix](https://remix.run) is an open source React-based framework maintained by [Shopify](https://shopify.com/). Our CX Framework for Remix supports fetching data and running actions for your Custom Storefront with Atama. This guide walks you through setting up your website so you can render experiences.
```

Comment:
nit: should "Custom Storefront" be lowercase? It's not a defined term that means anything specific, right?

Reply:
Yes that makes sense. I also changed it in the hydrogen docs.